staging/src/k8s.io/cri-client/pkg/logs/logs_test.go

```diff
@@ -214,7 +215,7 @@ func TestReadRotatedLog(t *testing.T) {
 	tmpDir := t.TempDir()
 	file, err := os.CreateTemp(tmpDir, "logfile")
 	if err != nil {
-		assert.NoErrorf(t, err, "unable to create temp file")
+		require.NoErrorf(t, err, "unable to create temp file")
```

Comment:
do you need `Errorf` here? Will `Error` suffice?

Also please search for require.Errorf around, I think those were used incorrectly in other places and would need to be fixed (maybe separate PR)

Reply:
/lgtm remove

Reply:
/remove-lgtm